modules/photo/include/opencv2/photo.hpp

```diff
@@ -328,7 +328,7 @@ CV_EXPORTS_W void denoise_TVL1(const std::vector<Mat>& observations,Mat& result,
 //! @addtogroup photo_hdr
 //! @{
 
-enum { LDR_SIZE = 256 };
+constexpr auto LDR_SIZE = 256;
```

Comment:
Why `auto` when the more obvious type to write is `int`?

It is even 1 symbol less to type. 🤣

Reply:
Thank you for your point, I forget to revert this definition. I'l fix it to use enum as before fixing.